### PR TITLE
Update release guide and contributing template

### PR DESCRIPTION
**Pull request by curquiza**
Following the @alallema feedback, I tried to improve our guides related to the release process 🙂

### CODE REVIEW

**Review by alallema — APPROVED** [review]
Looks Great! Thank @curquiza

File: guides/integration-release.md [hunk 1]
```diff
@@ -18,19 +18,21 @@ Don't do a release without any usage purpose, for example for the update of a te
 
 Follow the steps in the `Release Process` section in the `CONTRIBUTING.md` file of the repository carefully. These steps should be precise; if it's not, please open an issue on the related repository.
 
-During these steps, you might publish a release via the GitHub interface and so provide a description with the release changelogs.<br>
-The [Release Drafter tool](./release-drafter.md), integrated into most repositories, helps you provide a kind of structure, but please, feel free to add final modifications to ensure these changelogs are:
+#### Writing the Release Description
+
+During these release steps, you might publish a release via the GitHub interface and so provide a description with the release changelogs.<br>
```
Comment (bidoubiwa) [inline guides/integration-release.md:23]: If the release is dont not via the GitHub interface, you cannot provide a description with the changelogs or is there no description?

Not sure I understand

File: guides/integration-release.md [hunk 2]
```diff
@@ -18,19 +18,21 @@ Don't do a release without any usage purpose, for example for the update of a te
...
+The [Release Drafter tool](./release-drafter.md), integrated into most repositories, helps you provide a kind of structure, but please, feel free to add final modifications in the release description to ensure these changelogs are:
```
Comment (bidoubiwa) [inline guides/integration-release.md:24]:
```suggestion
The [Release Drafter tool](./release-drafter.md), integrated into most repositories, provides a release description structure with all pull requests added in this version (__without `skip-changelog` label__). Nonetheless, please add additional content in the release description to ensure these changelogs are:
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

File: guides/integration-release.md (hunk 1 above)
Comment (curquiza) [reply guides/integration-release.md:23]: Not sure to understand your question 😂

Comment (bidoubiwa) [reply guides/integration-release.md:23]: 😂 
> you might publish a release via the GitHub interface and so provide a description with the release changelogs.

Can you not provide a description using de github CLI?

Comment (curquiza) [reply guides/integration-release.md:23]: This process is not for releases that don't use the GH interface. The integrations that need to release in a different way already have a really detailed section in their CONTRIBUTING (the devops integration repositories for example).

Comment (curquiza) [reply guides/integration-release.md:23]: Only the repositories that need to release to the GH interface link this guide in their CONTRIBUTING

Comment (bidoubiwa) [reply guides/integration-release.md:23]:
```suggestion
During these release steps, depending on the integration, releasing is done via the GitHub interface. The release comes with a description containing the changelogs. <br>
```

**Comment by curquiza** [comment]
Thanks for your feedback! Merging!

**Pull request merged**